src/components/component.ts

```diff
@@ -163,7 +163,16 @@ module Plottable {
       };
     }
 
-    public render() {
+    /**
+     * Queues the Component for rendering.
+     * 
+     * @param immediate boolean Immediately renders the Component.
```

Comment:
"immediately" might be a better name for the param, and we probably don't need a description in that case.

Also need a @return